less/home-page.less

```diff
@@ -24,6 +26,10 @@
       margin: 20px 2.5%;
       border-radius: 20px;
       height: 100%;
+
+      @media @desktop{
+
```

Comment:
if you want to do mobile first, then you would need a desktop media query, but for this project, probably makes sense to do desktop first, so no desktop view is needed.  that's the default.